test/http-test.js

```diff
@@ -416,6 +416,26 @@ describe("httptest", () => {
 
       expect(resp.body).to.equal(JSON.stringify({ a: "b" }));
     });
+
+    it("content-type application/json returns body as json", async () => {
+      const resp = await request(app)
+        .post("/api")
+        .send({ a: "b" })
+        .expect("content-type", "application/json; charset=utf-8")
+        .expect(201);
+
+      expect(resp.body).to.deep.equal({ a: "b" });
+    });
+
+    it("content-type application/json with text body throws", async () => {
+      const resp = await request(app)
+        .post("/api")
+        .send({ a: "b" })
+        .expect("content-type", "application/json; charset=utf-8")
+        .expect(201);
+
+      expect(resp.body).to.deep.equal({ a: "b" });
+    });
```

Comment:
does description actually match assertions? seems identical to previous test

Reply:
No it doesn't, sorry.